README.rst

```diff
@@ -46,9 +46,9 @@ Project details
 * Manual page: man ubuntu-image
   (https://github.com/CanonicalLtd/ubuntu-image/blob/master/ubuntu-image.rst)
 
-The ``gadget.yaml`` specification has moved to `the snapcore repository`_.
+The ``gadget.yaml`` specification has moved to `the snapcraft forum`_.
 
-.. _`the snapcore repository`: https://github.com/snapcore/snapd/wiki/Gadget-snap
+.. _`the snapcraft forum`: https://forum.snapcraft.io/t/the-gadget-snap/696
```

Comment:
I wonder if we should simply link to https://forum.snapcraft.io/t/the-gadget-snap which seems to redirect to the proper forum post anyway.

Reply:
A brief check reveals that the number is the unique identifier of the topic in Discourse. A topic with a duplicate title could be an issue, depending on how Discourse is configured, as could renaming the topic. Renaming the topic seems like the most likely scenario that would cause the link to break in the future and as long as the identifier is there, Discourse will present the appropriate topic whatever it may have been renamed to. I'm all for prettier URL's, but I'm leaning towards keeping the ID there since things tend to get refactored and renamed often enough to potentially break links like this.

The official documentation doesn't seem to say much on this at a glance, but many posts recommend keeping the topic's identifier in the URL, such as https://meta.discourse.org/t/will-changing-a-topics-title-break-a-link-to-the-topic/131719.

Your thoughts?